fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -295,6 +342,21 @@ if [ ! -e "$fMRIFolder" ] ; then
 fi
 cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz
 
+# --- Do slice time correction if indicated
+if [ $DoSliceTimeCorrection = "TRUE" ] ; then
+    log_Msg "Running slice timing correction"
+    #  use FSL's fslreorient2std for reorienting the image to match the approximate orientation of the standard template images MNI152
+    #  this makes the single-band processing more robust to registration problems    
```

Comment:
This won't work.  You can't reorient the slice acquisition axis without also accounting for that as part of STC.  I see that `slicetimer` has an option (`-d, --direction`) to set the direction of slice acquisition, but if you want to use that, you will need to add code that ascertains what is the axes of the slice acquisition after `fslreorient2std` has been applied.  (If you have previously processed any non-axially acquired data using this approach, then the STC applied to that data is completely wrong).

Reply:
Is it ok if we do reorient after slice timing correction?

Reply:
As regards STC itself, yes, but not in the broader context of `fMRIVolume` overall, where:
1) changing the orientation changes the UnwarpDir.
2) I'm not immediately sure of the implications on the SEFMs either.

This is a complicated issue if you want the reorientation to be part of the pipeline code itself.

Reply:
Hmm. What if we currently limit it to only those data that do not have any DC method available?

Reply:
As this is quite common intersection in legacy datasets with SB bolds with no DC, so it would cover most cases where fslreorient might help.

Reply:
I _think_ that would prevent an inappropriate application, but I'd like @coalsont and @glasserm to chime in that I'm not missing some other potential caveat.  Reorienting your data is a tricky issue, because as you can see, numerous things are framed around the orientation of the data.

Reply:
I think the resampling into standard space uses an isoxform on the MNI T1 atlas image to generate the reference, so the output will still be standard orientation without an explicit reorient.

What is the reason for reorienting closer to the input stage?

Reply:
Oh, right, it's FSL with their ANALYZE-type coordinate system that ignores the nifti orientation and origin info.  So, the initialization for the registration might be at completely the wrong angle.

I wonder how this is handled in the NHP modifications.

Reply:
I don't think we do any reorientation for the _fMRI_ data.  We do for the T1w and T2w inputs, in PreFS, and then we comment in `T2wToT1wDistortionCorrectAndReg` that `--unwarpdir` is "post fslreorient2std".  But I think we've always just assumed that the fMRI data would be axially acquired (which is by far and away the most common).

Reply:
If it is axially acquired, then dimension 3 won't become dimension 1 or 2 when reoriented to standard, the worst that could happen is it is flipped.

Reply:
Chad and Takuya said they hand-reoriented (sometimes needing to relabel the axes) the NHP inputs before using them, so the NHP pipelines never need to deal with non-axial data.

Reply:
It is probably okay, though we should think about how this would interact with fieldmap-less distortion correction if it is implemented.

Reply:
Can I assume the consensus is we limit it to the data w/o available DC correction method?

Reply:
For now, I think it is either that, or you eliminate the `fslreorient2std` part entirely.  Regardless, people need to be aware of the reorienting, so I'd include a prominent comment at that section of the code, and also issue a `log_Warn` on the issue.

Reply:
Did that. @mharms can you please check the note and warning and suggest changes if necessary.